PlayerMatcher/Matchmaker/Matchmaker.cs

```diff
@@ -22,19 +22,20 @@ public List<User> ConstructMatch(int gameID, int numPlayers)
             int numTaken = 0;
             int minElo = 0;
             int maxElo = 10000;
+            int maxRuns = 100;
```

Comment:
so this is one way to accommodate not finding player. maybe another, possibly better way would be to first ensure that there are SOME players between the max and min range with a  single query, and then accelerate (increment) the window increase if you find none each iteration... just a thought.